### ISSUE TITLE
rendered Documentation updates

### ISSUE THREAD

**Issue by xmatthias** [ccxt/ccxt#15260]
- CCXT version: latest (not really relevant)

it seems like the documentation under https://docs.ccxt.com/en/latest/ is not being updated correctly.

Looking at 8dd987c0bfe7a6c4eb4c12b1a3805cfcf567ed3d (one of the latter commits that update the documentation) - we can see that the "Open Interest" section is missing from the rendered docs.

![image](https://user-images.githubusercontent.com/5024695/195333439-558be48e-6d01-47ad-8827-80ca46f297e0.png)

Looking at https://readthedocs.org/projects/ccxt/builds/ makes it clear that the documentation building is failing since a few weeks..
The last successful build (as far as i could find) dates from [1 month+3 weeks ago](https://readthedocs.org/projects/ccxt/builds/17814638/) - and can currently be found on page 57 of the above link (it's probably page 58 or 59 pretty soon).

Is there a (rendered) alternative to docs.ccxt.com which is currently updated correctly?

**Comment by Dan-krm** [comment]
> Looking at [8dd987c](https://github.com/ccxt/ccxt/commit/8dd987c0bfe7a6c4eb4c12b1a3805cfcf567ed3d) (one of the latter commits that update the documentation) - we can see that the "Open Interest" section is missing from the rendered docs.

I think I edited the wrong manual file in the Okx PR that you mentioned. I attempted to adjust the correct manual file in this PR: #15254 it looks like that hasn't worked yet though. Hopefully some other CCXT team members can provide some insight because I'm currently unsure of what the root of the issue is

**Comment by xmatthias** [comment]
It's not pinned onto that commit specifically.

https://readthedocs.org/projects/ccxt/builds/ shows that the build is failing with the below error - which seems to be a generic error (maybe caused by some dependency update without pinned version ...)

![image](https://user-images.githubusercontent.com/5024695/195516326-be9c4d97-dd7b-4a14-885c-cf63c14580d3.png)

**Comment by ttodua** [comment]
> Is there a (rendered) alternative to docs.ccxt.com which is currently updated correctly?

if you need a quick review, as you might already know, you can refer to : https://github.com/ccxt/ccxt/wiki/Manual however, the rendered manuals (as I think) is available only after successful build of that. So, it needs to be fixed still.

**Comment by xmatthias** [comment]
@kroitor unfortunately, #15274 doesn't fully fix "this" issue (as now a new issue occured).

from here: https://readthedocs.org/projects/ccxt/builds/

![grafik](https://user-images.githubusercontent.com/5024695/202153067-c0c5bc0c-fb68-406b-985b-626358b5f8d5.png)

**Comment by xmatthias** [comment]
that's correct - i'm just pointing out that the "docs.ccxt.com" page is not uptodate (hasn't been since august) ...

**Comment by xmatthias** [comment]
Considering builds are passing again - i think we can close this now.
